addons/product_margin/tests/test_product_margin.py

```diff
@@ -40,6 +40,7 @@ def test_product_margin(self):
                 'invoice_line_ids': [(0, 0, {'product_id': ipad.id, 'quantity': 10.0, 'price_unit': 550.0})],
             },
         ])
+        invoices.invoice_date = invoices[0].date
```

Comment:
Force `invoice_date` and `date` at an arbitrary value instead

Reply:
I would prefer keeping exactly the same behavior to minimize risks of breaking something subtle